### PR TITLE
Add UserContext

### PR DESCRIPTION
**Pull request by mojo9492**

### CODE REVIEW

**Comment by mojo9492** [comment]
The issue with prettier seems to have been resolved

File: client/src/Users/UserApi.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ export interface TsrUser {
 }
 
 export const emptyTsrUser: TsrUser = {
+    // TODO shouldn't need
```
Comment (gorhack) [inline client/src/Users/UserApi.ts:44]: do we still need this??

**Review by gorhack — CHANGES_REQUESTED** [review]
Please remove the org information from the drawer menu and the empty tsr user.

**Comment by mojo9492** [comment]
Ready @gorhack

**Review by gorhack — APPROVED** [review]

**Pull request merged**